src/Application/Migrations/06_tickets.sql

```diff
@@ -145,7 +152,7 @@ CREATE UNIQUE INDEX unique_fahrplan_id ON tbl_ticket (project_id, fahrplan_id) W
 -- trigger
 CREATE TRIGGER valid_handle BEFORE INSERT OR UPDATE ON tbl_ticket FOR EACH ROW EXECUTE PROCEDURE valid_handle();
 CREATE TRIGGER state_trigger BEFORE INSERT OR UPDATE OF ticket_state ON tbl_ticket FOR EACH ROW EXECUTE PROCEDURE update_ticket_next_state();
-CREATE TRIGGER progress_trigger1 AFTER INSERT OR UPDATE OF ticket_state ON tbl_ticket FOR EACH ROW EXECUTE PROCEDURE update_ticket_progress();
+CREATE TRIGGER progress_trigger1 AFTER INSERT OR DELETE OR UPDATE OF ticket_state ON tbl_ticket FOR EACH ROW EXECUTE PROCEDURE update_ticket_progress();
```

Comment:
Also: Although it shouldn't happen with the current feature set in the UI, but technically I could alter the parent_id of a ticket and this trigger wouldn't be fired, since it only matches "ON UPDATE OF ticket_state", right?

Reply:
Yes, altering parent_id is no use case for this trigger. As DELETE hasn't been previously as it seems ;)

Reply:
Is not but should be, right?

Reply:
As you stated, not possible with UI, and I am not aware of an API method that does this. But right, it doesnt hurt either.